### ISSUE TITLE
Crash when I try to pick up a high-rare shovel

### ISSUE THREAD

**Issue by Ryojing** [binary404/MysticTools-2.0#20]
Hello. Crash occurs everytime I try to pick up a purple named shovel.
Here is crash report.(I am poor at English. I apologize if these sentences are wrong.)

[A crash-2023-02-02_20.19.53-client.txt](https://github.com/binary404/MysticTools/files/10568521/A.crash-2023-02-02_20.19.53-client.txt)

**Comment by Ryojing** [comment]
Unique pickaxe and Unique axe also cause crash

**Comment by binary404** [comment]
Did you get those items from the /generateLoot command?

**Comment by Ryojing** [comment]
No, I didn't. But I used "Shovel Case"(Pickaxe Case or Axe Case) in Creative mode inventory at that time.
Is that problem whether I used command？
If crash doesn't occur in Survival mode,actually
it is really no problem for me.
